app/public/src/pages/component/profile/history.tsx

```diff
@@ -43,6 +62,12 @@ export default function History(props: { history: IGameRecord[] }) {
             </div>
           ))}
       </div>
+      <button
+          onClick={loadMore}
+          className="bubbly green create-room-button"
```

Comment:
wrong classname

Reply:
ya because this is a new button I created and it doesn't have any css.
should I create a new className for that button with the same css?

Reply:
bubbly green should be enough but create-room-button, definitely not appropriate